spec/integrational/protractor-config/bug55.js

```diff
@@ -4,7 +4,6 @@ exports.config = {
   seleniumAddress: env.seleniumAddress,
   framework: 'jasmine2',
   specs: ['../protractor/angularjs-homepage-disabled-test.js'],
-  useBlockingProxy: true,
```

Comment:
Somehow this causes tests fail (protractor raises an error `Error from wait for angular`).
This isn't specified in `default.js` and `useBlockingProxy` looks have nothing to do with the purpose of `bug55`, and I simply remove that.